tests/pykafka/test_simpleconsumer.py

```diff
@@ -228,16 +228,28 @@ def test_partition_saves_offset(self):
         self.assertNotEqual(ret_message, None)
         self.assertEqual(ret_message.value, msgval)
 
-    def test_partition_rejects_old_message(self):
+    def test_partition_rejects_old_message_after_initial(self):
```

Comment:
If we're going with a kwarg that indicates that a topic is compacted, I think a test should be added here, not altered.